Example/EpoxyExample/ViewControllers/LayoutGroups/ColorsViewController.swift

```diff
@@ -0,0 +1,46 @@
+// Created by Tyler Hedrick on 1/27/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCollectionView
+import UIKit
+
+class ColorsViewController: CollectionViewController {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(layout: UICollectionViewCompositionalLayout.list)
+    setItems(items, animated: false)
+  }
+
+  // MARK: Internal
+
+  enum DataID {
+    case hGroupFill
+    case hGroupTop
+    case hGroupCenter
+    case hGroupBottom
+    case vGroupFill
+    case vGroupLeading
+    case vGroupCenter
+    case vGroupTrailing
+  }
+
+  override var title: String? {
+    get { "Static sized squares" }
+    set { }
+  }
+
+  var items: [ItemModeling] {
```

Comment:
nit: using an `@ItemModelBuilder` here will reduce indentation.
```suggestion
  @ItemModelBuilder
  var items: [ItemModeling] {
```